src/style.css

```diff
@@ -1,3 +1,70 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1005px;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo-link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.menu-list {
+  display: flex;
+}
+
+.nav__item {
+  color: black;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.item-link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin: auto;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 500;
+}
+
+.active__color {
+  color: #00acdc;
```

Comment:
```suggestion
  color: var(--main-color);
```

Reply:
Not fixed, you need to create variable for repeated color and use it everywhere 
<img width="875" alt="image" src="https://github.com/user-attachments/assets/e6bff692-4a70-4113-8e76-6865348dfe56" />